### PR TITLE
additional resources

### PR DESCRIPTION
**Pull request by aviv**
cf. #143, #144, #114

PRing for feedback on the concept, and to discuss what content we actually want here.

When we have content, we can make these separate pages:

## Good forms

- Links to external content on making clear, effective forms?

- A "form guidelines" resource that I can write- could include when to use each field type, general rules, etc. Should be based on how people are currently using Screendoor, and how they can use it more effectively to build better forms. If we're going to make this a useful resource, we should probably do some brainstorming first about what to focus on.

## Good content

Somehow this seems a bit patronizing to me. Is the goal here to help people learn how to write properly? I.e. The knowledge base should be a resource for learning how to use our apps, and this seems like it might go too far beyond that.

/cc @dobtco/team if anyone has any ideas about what kind of content to include, feel free to comment.

### CODE REVIEW

**Comment by ajb** [comment]
"Good forms" looks great.

@cjoh (and @jrubenoff of course) will probably want to chime in on "Good content". I don't think it's patronizing to remind people that thought they might be used to writing in government language, Screendoor is a place that makes their projects more accessible, and as such, they need to use clear language.

Minor implementation issues:

- Replace "Adopta" on homepage with "additional resources" http://take.ms/eUJtC
- The text of this option should be empty, and it should be the first option in the list: http://take.ms/DPPvC
- Additional resources should be an item in the dropdown: http://take.ms/YHNSq

**Comment by aviv** [comment]
I had thought we didn't want to put "Additional resources" in that same dropdown in order to keep it separate. But this works too.

**Comment by ajb** [comment]
Yeah I think that works well :+1:

**Comment by kariannemah** [comment]
Under "Good forms" I'd consider linking to [Project Streamline](http://www.gmnetwork.org/projectstreamline/publications/champions-toolkit/). Though geared toward grantmakers, it has useful advice for anyone asking for information from another person. The focus is on "right-sizing" applications (i.e., making application and reporting requirements proportionate to the size and type of grant/contract).

**Comment by ajb** [comment]
@aviv can we pick this up this week? I'd like to also do a blogpost on it once it's done... 

Oakland's current grant applications are pretty horrendous. I want to start brainstorming larger-picture solutions.

File: contents/articles/additional_resources/content/good_forms.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+<form class='example_fr fr_no' id='example_fr_5'></form>
+
+Always include clear labels:
+
+<form class='example_fr fr_yes' id='example_fr_6'></form>
+
+## Need more help?
+
+Let us help you make your form awesome. We're available for consultation on an hourly basis.
```
Comment (aviv) [inline contents/articles/additional_resources/content/good_forms.md]: do we want them to just email support@dobt.co for now?

**Comment by aviv** [comment]
@ajb take a look and tell me if I'm on the right track?

File: contents/articles/additional_resources/content/good_forms.md (hunk 1 above)
Comment (ajb) [inline contents/articles/additional_resources/content/good_forms.md]: Yeah that's fine.

File: contents/js/main.coffee [hunk 2]
```diff
@@ -1,3 +1,10 @@
+# Copied form dvl-core
```
Comment (ajb) [inline contents/js/main.coffee]: Maybe we can figure out how to reference these from nap

**Comment by ajb** [comment]
:+1:, pinging @jrubenoff for feedback too.

What about some of the tips at the bottom of this doc? https://docs.google.com/document/d/1u-ylcuhPvzWEr2mWZy8oMo-UEzsxx07oEzxMgleXNiE/edit#

File: contents/js/main.coffee (hunk 2 above)
Comment (aviv) [inline contents/js/main.coffee]: This doesn't work:

from `config.json`:
```json
  "nap": {
    "assets": {
      "js": {
        "dvl": [
          "/vendor/assets/js/dvl/core/styled_controls.coffee"
        ],
        "main": [
          "/js/purl.js",
          "/js/jquery.placeholder.js",
          "/js/main.coffee"
        ]
      }
    }
  }
```

I still don't really understand how `dvl` is actually being loaded via `bundler`, i.e. how `@import 'vendor/assets/stylesheets/dvl/components/splash_footer';` is working. It still feels super hacky to be loading `dvl` via `bundler` in the first place.

Comment (ajb) [inline contents/js/main.coffee]: Let me take a shot at it later.

On Wed, Apr 29, 2015 at 2:18 PM, Aviv Nitsan <notifications@github.com>
wrote:

> In contents/js/main.coffee
> <https://github.com/dobtco/knowledge_base/pull/168#discussion_r29364258>:
>
> > @@ -1,3 +1,10 @@
> > +# Copied form dvl-core
>
> This doesn't work:
>
> from config.json:
>
>   "nap": {
>     "assets": {
>       "js": {
>         "dvl": [
>           "/vendor/assets/js/dvl/core/styled_controls.coffee"
>         ],
>         "main": [
>           "/js/purl.js",
>           "/js/jquery.placeholder.js",
>           "/js/main.coffee"
>         ]
>       }
>     }
>   }
>
> I still don't really understand how dvl is actually being loaded via
> bundler, i.e. how @import
> 'vendor/assets/stylesheets/dvl/components/splash_footer'; is working. It
> still feels super hacky to be loading dvl via bundler in the first place.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/knowledge_base/pull/168/files#r29364258>.
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker

**Comment by aviv** [comment]
The tips in that doc seem to be mostly "general ideas" rather than "quick tips"; they also might fit better in "Good content" rather than "Good forms." Added a sentence based on "Group related information together" from that doc, though.

**Comment by ajb** [comment]
Well "good content" is part of this PR, no? ;)

On Wed, Apr 29, 2015 at 2:25 PM, Aviv Nitsan <notifications@github.com>
wrote:

> The tips in that doc seem to be mostly "general ideas" rather than "quick
> tips"; they also might fit better in "Good content" rather than "Good
> forms." Added a sentence based on "Group related information together" from
> that doc, though.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/knowledge_base/pull/168#issuecomment-97531928>.
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker

File: contents/articles/additional_resources/content/good_content.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
```
Comment (jrubenoff) [inline contents/articles/additional_resources/content/good_content.md]: I would add this to the bottom of the "Good forms" page, if we only have links here right now.

Comment (aviv) [inline contents/articles/additional_resources/content/good_content.md]: Hmm, but the "Good forms" page is already pretty long. My feeling is that we should keep the specifically form-oriented stuff separate. The "Good content" stuff is (and when we add more stuff, will be) way more general.

**Comment by aviv** [comment]
OK, the "Good content" page is pretty rough, but maybe we can think of this as a resource we can reference and add to as needed rather than a finished product. I.e. it might make more sense to add stuff as it comes up rather than trying to brainstorm a coherent list before publishing.

**Comment by ajb** [comment]
How about combining the "good content" and "good forms" pages for now? Reasons:

- "Good content" looks incomplete as-is
- Our content tips mostly apply to form copy.
- We're a forms company, not a content company

**Comment by aviv** [comment]
Sounds good, but then I think we should eliminate the "additional
resources" section altogether and simply fit the "good forms" article
into one of the existing screendoor sections.

**Comment by ajb** [comment]
Sounds great. Let's ship today.

On Fri, May 1, 2015 at 11:43 AM, Aviv Nitsan <notifications@github.com>
wrote:

> Sounds good, but then I think we should eliminate the "additional
> resources" section altogether and simply fit the "good forms" article
> into one of the existing screendoor sections.
>
> On Fri, May 1, 2015, at 07:39 AM, Adam Becker wrote:
> > How about combining the "good content" and "good forms" pages for now?
> > Reasons:
> >
> > - "Good content" looks incomplete as-is
> > - Our content tips mostly apply to form copy.
> > - We're a forms company, not a content company
> >
> >
> > ---
> > Reply to this email directly or view it on GitHub:
> > https://github.com/dobtco/knowledge_base/pull/168#issuecomment-98146215
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dobtco/knowledge_base/pull/168#issuecomment-98161279>.
>



-- 
Adam Becker
(951) 9-BECKER
@AdamJacobBecker

**Comment by aviv** [comment]
Here's how I re-arranged the sections:



Assigning to @kariannemah for final review.

**Comment by kariannemah** [comment]
:+1: Looks great! 

Made small edits, not too many, mostly commas: :smile:

**Comment by aviv** [comment]
:ship: :zap:

**Pull request merged**